app/templates/custom-elements/remote-screen.html

```diff
@@ -199,6 +190,107 @@
           screenElement.addEventListener("mousedown", (evt) => {
             this.rateLimitedMouse.onMouseDown(evt);
           });
+          //-------------------------------------------------------------------------------
+          var isPointerMoving = false;
+          var touchStartTimestamp = null;
+          var longTouchDuration = 600; // Time in milliseconds for a touch to be considered a long touch
+          var touchTimeoutId = null;
+          let touchClientX = null;
+          let touchClientY = null;
+          //listens to touchstart and dispatches a right click event if the touch duration is longetr than 1000ms
+          screenElement.addEventListener("touchstart", (evt) => {
+            if (browser == "safari" || browser == "firefox")
+              //evt needs to disabled only on firefox and safari to avoid default long press action
+              evt.preventDefault();
+            touchStartTimestamp = Date.now();
+            touchClientX = evt.touches[0].clientX;
+            touchClientY = evt.touches[0].clientY;
+            // touchTimeoutId will be triggered only if the touch duration is longer than 600ms
+            touchTimeoutId = setTimeout(function () {
+              if (!isPointerMoving) {
+                let mouseEvent = new MouseEvent("mousedown", {
+                  bubbles: true,
+                  cancelable: true,
+                  view: window,
+                  button: 0,
+                  buttons: 2,
+                  clientX: evt.touches[0].clientX,
+                  clientY: evt.touches[0].clientY,
+                });
+                screenElement.dispatchEvent(mouseEvent);
+              }
+            }, longTouchDuration);
+          });
+
+          screenElement.addEventListener("touchend", (evt) => {
+            evt.preventDefault();
+            isPointerMoving = false;
+            var touchEndTimestamp = Date.now();
+            var touchDuration = touchEndTimestamp - touchStartTimestamp;
+            //if touch duration is less than 600ms it is considered as a normal touch
+            //and left left button down is dispatched which is followed by mouseup
+            if (touchDuration < longTouchDuration && touchTimeoutId !== null) {
+              clearTimeout(touchTimeoutId);
+              //Normal touch"
+              //if normal touch dispatching mousedown event
+              let mouseEvent = new MouseEvent("mousedown", {
+                bubbles: true,
+                cancelable: true,
+                view: window,
+                button: 0,
+                buttons: 1,
+                clientX: touchClientX,
+                clientY: touchClientY,
+              });
+              screenElement.dispatchEvent(mouseEvent);
+              //and an immediate mouse up event.
```

Comment:
Can we keep the comments full sentences and sentence-style capitalization? Also, our convention is to include a single space after the `//`.

(ditto throughout)